packages/labs/compiler/package.json

```diff
@@ -44,10 +45,26 @@
       ],
       "clean": "if-file-deleted"
     },
+    "build:rollup-source-map-tests": {
+      "command": "rollup -c rollup.source_map_tests.js",
+      "dependencies": [
+        "build"
```

Comment:
should this depend on `build:ts`, and `build` actually have `build:rollup-source-map-tests` as a dependency so that `npm run build` builds everything?

Reply:
Thank you! I think I've fixed this.

`build` -> builds everything.
`build:ts` -> only builds typescript.
`build:rollup-source-map-tests` -> depends on build:ts.
`test` -> depends on `build`.